src/components/chat-sidebar/chat-sidebar-link.tsx

```diff
@@ -14,7 +14,7 @@ export function ChatSidebarLink({
     <SidebarMenuItem>
       <SidebarMenuButton asChild>
         <Link
-          href={`chats/${uuid}`}
+          href={`/chats/${uuid}`}
```

Comment:
```suggestion
          href={`/chat/${uuid}`}
```
Our route is `/chat/[uuid]` - without the S. After you fix this, you can go ahead and merge

Reply:
oof, my bad XD